packages/ui/react-ui-mosaic/src/mosaic/Tile.tsx

```diff
@@ -104,37 +104,34 @@ export const DraggableTile = ({
 /**
  * Basic droppable mosaic tile.
  */
-export const DroppableTile = ({
-  path: parentPath,
-  type = DEFAULT_TYPE,
-  item,
-  Component = DefaultComponent,
-  position,
-  ...props
-}: MosaicTileProps<any>) => {
-  const { operation } = useMosaic();
-  const path =
-    parentPath === item.id
-      ? parentPath // If the path is the same as the item id, then this is the root tile.
-      : Path.create(parentPath, item.id);
-  const { setNodeRef, isOver } = useDroppable({
-    id: path,
-    data: { path, type, item, position } satisfies MosaicDraggedItem,
-  });
-
-  return (
-    <Component
-      ref={setNodeRef}
-      item={item}
-      path={path}
-      type={type}
-      position={position}
-      operation={operation}
-      isOver={isOver}
-      {...props}
-    />
-  );
-};
+export const DroppableTile = forwardRef<HTMLDivElement, MosaicTileProps<any>>(
```

Comment:
Elsewhere in mosaic I think I used `HTMLElement` for the generic ref type at the api level to allow it to be cast to any element inside the component

Reply:
Right, I was experimenting with ref passing and intended to polish this up before merging. It’s no longer necessary, so I'll revert this.

Reply:
Reverted 👍